config/secret_config.go

```diff
@@ -0,0 +1,99 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/jahvon/tuikit/types"
+	"gopkg.in/yaml.v3"
+)
+
+type SecretConfig struct {
+	Name   string `json:"name"  yaml:"name"`
+	Secret string `json:"value" yaml:"value"`
+}
+
+type SecretConfigList []SecretConfig
+
+type enrichedSecretConfigList struct {
+	Secrets SecretConfigList `json:"secrets" yaml:"secrets"`
+}
+
+func (c *SecretConfig) AssignedName() string {
+	return c.Name
+}
+
+func (c *SecretConfig) SecretValue() string {
+	return c.Secret
+}
+
+func (c *SecretConfig) YAML() (string, error) {
+	yamlBytes, err := yaml.Marshal(c)
+	if err != nil {
+		return "", fmt.Errorf("failed to marshal secret config - %w", err)
+	}
+	return string(yamlBytes), nil
+}
+
+func (c *SecretConfig) JSON() (string, error) {
+	jsonBytes, err := json.MarshalIndent(c, "", "  ")
+	if err != nil {
+		return "", fmt.Errorf("failed to marshal secret config - %w", err)
+	}
+	return string(jsonBytes), nil
+}
+
+func (c *SecretConfig) Markdown() string {
+	var mkdwn string
+	mkdwn = fmt.Sprintf("# [Secret] %s\n", c.AssignedName())
+	mkdwn += fmt.Sprintf("## Value\n%s\n", c.SecretValue())
```

Comment:
```suggestion
	mkdwn += fmt.Sprintf("**Value**\n```\n%s\n```", c.SecretValue())
```
Let's wrap the value in a codeblock

Reply:
Sure